streams/src/main/java/org/apache/kafka/streams/state/internals/CachingKeyValueStore.java

```diff
@@ -228,6 +232,12 @@ public void putAll(final List<KeyValue<Bytes, byte[]>> entries) {
     @Override
     public KeyValueIterator<Bytes, byte[]> range(final Bytes from,
                                                  final Bytes to) {
+        // Make sure this is a valid query
```

Comment:
nit: I would remove the comment here (and in all occurrences below), because the code itself is clear enough about what it does. Maybe rename `from` and `to` to `fromKey` and `toKey` (or similar) to make it even more clearer. Renaming would also apply to some of the changes below.

Reply:
Ack